pkg/gui/controllers/toggle_whitespace_action.go

```diff
@@ -23,7 +23,8 @@ func (self *ToggleWhitespaceAction) Call() error {
 		return self.c.ErrorMsg(self.c.Tr.IgnoreWhitespaceNotSupportedHere)
 	}
 
-	self.c.State().SetIgnoreWhitespaceInDiffView(!self.c.State().GetIgnoreWhitespaceInDiffView())
+	self.c.GetAppState().IgnoreWhitespaceInDiffView = !self.c.GetAppState().IgnoreWhitespaceInDiffView
+	_ = self.c.SaveAppState()
```

Comment:
could we log the error (`self.c.Log.Error`) here

Reply:
Sure: d11af0992a03. Since I copied the code from the show-hide command log handler, where we don't log it either, I added a commit to log it there, too (fb7e2685515c).